examples/with_custom_redis_pool.rs

```diff
@@ -0,0 +1,84 @@
+extern crate spaceapi_server;
+extern crate env_logger;
+extern crate redis;
+extern crate r2d2;
+extern crate r2d2_redis;
+
+use spaceapi_server::SpaceapiServerBuilder;
+use spaceapi_server::api;
+use spaceapi_server::modifiers::StatusModifier;
+
+use r2d2::Pool;
+use r2d2_redis::RedisConnectionManager;
+use redis::{Commands, RedisResult};
+
+type RedisPool = Pool<RedisConnectionManager>;
+
+struct OpenStatusFromRedisModifier {
+    pool: RedisPool
+}
+
+impl OpenStatusFromRedisModifier {
+    fn new(pool: RedisPool) -> OpenStatusFromRedisModifier {
+        OpenStatusFromRedisModifier{pool}
+    }
+}
+
+impl StatusModifier for OpenStatusFromRedisModifier {
+    fn modify(&self, status: &mut api::Status) {
+        let conn = self.pool.get().unwrap();
+        let state : RedisResult<String> = conn.get("state_open");
+        status.state.open = match state {
+            Ok(v) => Some(v == "open"),
+            Err(_) => None
+        };
+        status.state.lastchange = conn.get("state_lastchange").ok();
+        status.state.trigger_person = conn.get("state_triggerperson").ok();
+    }
+}
+
+fn main() {
+    env_logger::init().unwrap();
+
+    // Create new minimal Status instance
```

Comment:
The comment seems a bit off, since it is not a minimal example, right? We can also just remove the comment IMO.

Reply:
Oh, right! definitely a copy-paste error :)